### ISSUE TITLE
Failed to open collectd RRD database

### ISSUE THREAD

**Issue by bunder** [binjr/binjr#102]
binjr 2.17.0
OS: ArchLinux x86_64

When I try to add RRD database from `/var/lib/collectd/hostname/cpu-0/percent-idle.rrd` it fails with following exception:

```
eu.binjr.core.data.exceptions.DataAdapterException: Failed to open rrd db
	at eu.binjr.sources.rrd4j.adapters.Rrd4jFileAdapter.getBindingTree(Rrd4jFileAdapter.java:139)
	at eu.binjr.core.controllers.MainViewController.buildTreeViewForTarget(MainViewController.java:1099)
	at eu.binjr.core.controllers.MainViewController.lambda$addSource$37(MainViewController.java:771)
	at eu.binjr.core.data.async.AsyncTaskManager$1.call(AsyncTaskManager.java:83)
	at javafx.graphics/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
Caused by: java.nio.file.AccessDeniedException: /var/lib/collectd/vostrobook/cpu-0/percent-idle.rrd
	at java.base/sun.nio.fs.UnixException.translateToIOException(Unknown Source)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(Unknown Source)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(Unknown Source)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newFileChannel(Unknown Source)
	at java.base/java.nio.channels.FileChannel.open(Unknown Source)
	at org.rrd4j.core.RrdNioBackend.<init>(RrdNioBackend.java:91)
	at org.rrd4j.core.RrdNioBackendFactory.open(RrdNioBackendFactory.java:163)
	at org.rrd4j.core.RrdBackendFactory.getBackend(RrdBackendFactory.java:511)
	at org.rrd4j.core.RrdDb.<init>(RrdDb.java:612)
	at org.rrd4j.core.RrdDb.<init>(RrdDb.java:44)
	at org.rrd4j.core.RrdDb$Builder.build(RrdDb.java:93)
	at org.rrd4j.core.RrdDbPool.requestRrdDb(RrdDbPool.java:415)
	at org.rrd4j.core.RrdDbPool.requestRrdDb(RrdDbPool.java:366)
	at eu.binjr.sources.rrd4j.adapters.Rrd4jFileAdapter.getBindingTree(Rrd4jFileAdapter.java:108)
	... 11 more
```

Database is accessible by user:

```
$ file /var/lib/collectd/vostrobook/cpu-0/percent-idle.rrd 
/var/lib/collectd/vostrobook/cpu-0/percent-idle.rrd: RRDTool DB version 0003 64bit aligned little-endian 64bit long (alpha/amd64/ia64)
```

**Comment by fthevenet** [comment]
Hi,

I've reproduced the issue. It is actually caused by a regression introduced a couple of versions ago which prevents binjr from opening rrd db files generated with [RrdTools](https://github.com/oetiker/rrdtool-1.x).

If all goes well, I should be able to publish a fix shortly.

**Comment by fthevenet** [comment]
Fixed by c6fcdb4

**Comment by fthevenet** [comment]
I've published a snapshot version that includes the fix here: https://github.com/binjr/binjr/releases/tag/v2.17.1-SNAPSHOT

Could you please try it and let me know if it fixes the issue on your end? If so I'll publish a new release w/ the fix.

**Comment by bunder** [comment]
It works. Thank you for quick fix!

**Comment by fthevenet** [comment]
Released in [v2.17.1](https://github.com/binjr/binjr/releases/tag/v2.17.1)
